cuckoo/data-private/cwd/conf/esx.conf

```diff
@@ -56,4 +56,12 @@ resultserver_port = {{ esx[machine].resultserver_port }}
 # (Optional) Set your own tags. These are comma separated and help to identify
 # specific VMs. You can run samples on VMs with tag you require.
 tags = {{ esx[machine].tags }}
+
+# (Optional) Specify a memory profile to be used by volatility for this
+# virtual machine. This will override the guest_profile variable in
+# memory.conf which solves the problem of having multiple types of VM's
+# and properly determining which profile to use. Examples below:
+#mem_profile = WinXPSP2x86
+#mem_profile = Win7SP0x86
+#mem_profile = Win7SP1x64
```

Comment:
One of these should certainly be defined, i.e., `mem_profile = {{ esx[machines].mem_profile }}` etc as otherwise those values are missing from the configuration (which is an error).